stress/n_workers_reputers__stress_test.go

```diff
@@ -272,16 +273,135 @@ func InsertReputerBulk(m TestMetadata,
+const topicFunds int64 = 1000000
+const epochLength int64 = 5
+
+func SetupTopic(m TestMetadata, topicFunderAddress string, topicFunderAccount cosmosaccount.Account) uint64 {
+
+	m.t.Log("Creating new Topic")
+
+	topicId := CreateTopic(m, epochLength)
+
+	err := FundTopic(m, topicId, topicFunderAddress, topicFunderAccount, topicFunds)
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	err = RegisterWorkerForTopic(m, topicFunderAddress, topicFunderAccount, topicId)
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	err = RegisterReputerForTopic(m, topicFunderAddress, topicFunderAccount, topicId)
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	err = StakeReputer(m, topicId, topicFunderAddress, topicFunderAccount, stakeToAdd)
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	m.t.Log("Created new Topic with topicId", topicId)
+
+	return topicId
+}
+
+func WorkerReputerCoordinationLoop(m TestMetadata) {
+
+	reputersPerEpoch := lookupEnvInt(m, "REPUTERS_PER_EPOCH", 0)
+	reputersMax := lookupEnvInt(m, "REPUTERS_MAX", 10000)
+	workersPerEpoch := lookupEnvInt(m, "WORKERS_PER_EPOCH", 0)
+	workersMax := lookupEnvInt(m, "WORKERS_MAX", 10000)
+	topicsPerEpoch := lookupEnvInt(m, "TOPICS_PER_EPOCH", 0)
+	topicsMax := lookupEnvInt(m, "TOPICS_MAX", 100)
+
+	fmt.Println("Reputers per epoch: ", reputersPerEpoch)
+	fmt.Println("Reputers max: ", reputersMax)
+	fmt.Println("Workers per epoch: ", workersPerEpoch)
+	fmt.Println("Workers max: ", workersMax)
+	fmt.Println("Topics per epoch: ", topicsPerEpoch)
+	fmt.Println("Topics max: ", topicsMax)
+
 	approximateBlockTimeSeconds := getApproximateBlockTimeSeconds(m)
 	fmt.Println("Approximate block time seconds: ", approximateBlockTimeSeconds)
 
-	const stakeToAdd uint64 = 90000
-	const topicFunds int64 = 1000000
+	iterationTimeSeconds := time.Duration(epochLength) * approximateBlockTimeSeconds * iterationsInABatch * 2
+	topicCount := 0
+
+	getTopicFunder := func() (string, cosmosaccount.Account) {
+		topicFunderAccountName := "topicFunder" + strconv.Itoa(int(topicCount))
+		topicFunderAccount, _, err := m.n.Client.AccountRegistry.Create(topicFunderAccountName)
+		if err != nil {
+			fmt.Println("Error creating funder address: ", topicFunderAccountName, " - ", err)
+			return "", topicFunderAccount
+		}
+		topicFunderAddress, err := topicFunderAccount.Address(params.HumanCoinUnit)
+		if err != nil {
+			fmt.Println("Error getting funder address: ", topicFunderAccountName, " - ", err)
+			return topicFunderAddress, topicFunderAccount
+		}
+		err = fundAccount(m, m.n.FaucetAcc, m.n.FaucetAddr, topicFunderAddress, 100000000000000000)
+		if err != nil {
+			fmt.Println("Error funding funder address: ", topicFunderAddress, " - ", err)
+			return topicFunderAddress, topicFunderAccount
+		}
+		return topicFunderAddress, topicFunderAccount
+	}
+
+	if topicsPerEpoch == 0 {
+		topicFunderAddress, topicFunderAccount := getTopicFunder()
+		WorkerReputerLoop(m, topicFunderAddress, topicFunderAccount)
+		topicCount++
+	} else {
```

Comment:
same thing w the control flow here

Reply:
If we don't want to add any topics per epoch, then we just create a single topic by calling the `WorkerReputerLoop` 
 once. If we are adding topics every epoch, then we want to enter the loop to call `WorkerReputerLoop` repeatedly